src/components/Header/Header.tsx

```diff
@@ -3,6 +3,19 @@ import * as React from 'react'
 
 import { childrenExist, customPropTypes, UIComponent } from '../../lib'
 import HeaderDescription from './HeaderDescription'
+import { Extendable, ItemShorthand } from '../../../types/utils'
+import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
+
+export interface IHeaderProps {
+  as?: any
+  children?: React.ReactNode
```

Comment:
it seems that we are lying here, as it could be the case that several nodes are provided as children. We shouldn't make any assumptions that only one child node will be provided - otherwise we could accidentally introduce subtle problems in processing logic that won;t be caught by the compiler

Reply:
`React.ReactNodeArray | React.ReactNode` seems to be a desired alternative here

Reply:
Good catch, thanks a lot! I will provide in the utils.d.ts `Children` type and reuse it for the children prop.